docs/reference/commandline/cli.md

```diff
@@ -48,6 +48,8 @@ by the `docker` command line:
 * `DOCKER_TLS_VERIFY` When set Docker uses TLS and verifies the remote.
 * `DOCKER_CONTENT_TRUST` When set Docker uses notary to sign and verify images.
   Equates to `--disable-content-trust=false` for build, create, pull, push, run.
+* `DOCKER_CONTENT_TRUST_SERVER` The URL of the Notary server to use. This defaults
```

Comment:
Not sure we want to document this yet. It is purposely an undocumented flag b/c we don't yet want folks to depend on it. There will likely be a different way to configure notary servers in the future.